qa/L0_client/test.sh

```diff
@@ -50,6 +50,101 @@ for l in libgrpcclient.so libgrpcclient_static.a libhttpclient.so libhttpclient_
     fi
 done
 
+# Test a simple app using Triton gRPC API
+client_lib=$(pwd)/triton_client/lib
+
+# Test linking against the shared library
+rm simple_grpc_client
+g++ grpc_client.cc -o simple_grpc_client -Itriton_client/include \
+  -L$(pwd)/triton_client/lib -I/workspace/builddir/grpc/include \
+  -I/workspace/builddir/protobuf/include -lgrpcclient
+
+if [ $? -eq 0 ]; then
+    if [[ ! -x "./simple_grpc_client" ]]; then
+        echo -e "*** simple_grpc_client executable not present\n"
+        RET=1
+    else
+        ./simple_grpc_client
+        if [ $? -eq 0 ]; then
+            echo -e "\n***\n*** simple_grpc_client exited with 0 PASSED\n***"
+        else
+            echo -e "\n***\n*** simple_grpc_client exited with non-zero FAILED\n***"
+            RET=1
+        fi
+    fi
+else
+    echo -e "\n***\n*** Client headers build FAILED\n***"
+    RET=1
+fi
+
+static_libs="$client_lib/libgrpcclient_static.a $client_lib/libgrpc++.a $client_lib/libgrpc.a \
```

Comment:
Are you sure you need the libgrpc++.a and later static libraries on this line? From the cmakefile it seems that their content should be included in libgrpcclient_static.a

Reply:
Yes, that is very strange. I checked the symbols inside `libgrpcclient_static.a` using `nm` but it doesn't include the implementation of any of the libraries.

```
root@15095cf90c01:/workspace/qa/L0_client# nm --demangle triton_client/lib/libgrpcclient_static.a | grep "google::protobuf::io::ZeroCopyOutputStream::WriteAliasedRaw"
                 U google::protobuf::io::ZeroCopyOutputStream::WriteAliasedRaw(void const*, int)
                 U google::protobuf::io::ZeroCopyOutputStream::WriteAliasedRaw(void const*, int)
root@15095cf90c01:/workspace/qa/L0_client# nm --demangle triton_client/lib/libgrpcclient.so | grep "google::protobuf::io::ZeroCopyOutputStream::WriteAliasedRaw"
00000000002926b0 t google::protobuf::io::ZeroCopyOutputStream::WriteAliasedRaw(void const*, int)
```